src/index.tsx

```diff
@@ -43,6 +43,16 @@ let ensureTurnstile: () => Promise<any>;
   };
 }
 
+export function useTurnstile() {
+  const reset = () => {
+    global.turnstile.reset();
+  }
+
+  return {
+    reset
+  }
```

Comment:
Sample implementation:
```suggestion
  const [state, setState] = useState(turnstileState);
  useEffect(() => {
    if(turnstileState === "ready") return;
    turnstileLoad.then(() => setState(turnstileState))
  }, []);
  return global.turnstile;
```

Reply:
Yeah, I thought about including the other methods. That's why I structured the code the way I did. Good point about checking if SDK is loaded. I'll modify and resubmit.

Reply:
What is the 'state' for? It's not read anywhere in your example. Also I think an interval timer might be simpler to check when Turnstile is ready.

Reply:
Actually, now that I think of it, if `global.turnstile` is undefined, `useTurnstile` will return undefined anyway. Why would I need additional logic?

Reply:
`state` is used to trigger a component re-render, so the application actually receives the turnstile object once it's loaded